### PR TITLE
feat: classmodules@arcs

### PR DESCRIPTION
**Pull request by jacoobes**

### CODE REVIEW

**Comment by trueharuu** [comment]
hello

**Comment by jacoobes** [comment]
Hi

File: src/handler/utilities/readFile.ts [hunk 1]
```diff
@@ -73,9 +73,13 @@ export function buildData<T>(commandDir: string): Observable<
                 } catch {
                     mod = (await import(`file:///` + absPath)).default;
                 }
-                if (mod !== undefined) {
-                    return Ok({ mod, absPath });
-                } else return Err(SernError.UndefinedModule);
+                if(mod === undefined) {
```
Comment (EvolutionX-10) [inline src/handler/utilities/readFile.ts:76]: run prettier please

File: src/handler/structures/context.ts [hunk 2]
```diff
@@ -140,7 +140,7 @@ export default class Context {
                     .then(() => i.fetchReply());
             }),
             this.oMsg.map(m => {
-                return m.reply(content as string | ReplyMessageOptions);
+                return m.reply(content as string | MessageReplyOptions);
```
Comment (EvolutionX-10) [inline src/handler/structures/context.ts:143]: this fixes the 14.4 break for autocomplete right?

**Review by EvolutionX-10 — APPROVED** [review]
LGTM, hopefully tested properly!

File: src/handler/structures/context.ts (hunk 2 above)
Comment (jacoobes) [reply src/handler/structures/context.ts:143]: yes

**Pull request merged**